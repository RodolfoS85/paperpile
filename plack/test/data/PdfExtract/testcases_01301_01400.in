---
file: ../../../PDFREPOSITORY/001301.pdf
---
file: ../../../PDFREPOSITORY/001302.pdf
---
file: ../../../PDFREPOSITORY/001303.pdf
---
file: ../../../PDFREPOSITORY/001304.pdf
---
file: ../../../PDFREPOSITORY/001305.pdf
---
file: ../../../PDFREPOSITORY/001306.pdf
---
file: ../../../PDFREPOSITORY/001307.pdf
---
file: ../../../PDFREPOSITORY/001308.pdf
---
file: ../../../PDFREPOSITORY/001309.pdf
---
file: ../../../PDFREPOSITORY/001310.pdf
---
file: ../../../PDFREPOSITORY/001311.pdf
---
file: ../../../PDFREPOSITORY/001312.pdf
---
file: ../../../PDFREPOSITORY/001313.pdf
---
file: ../../../PDFREPOSITORY/001314.pdf
---
file: ../../../PDFREPOSITORY/001315.pdf
---
file: ../../../PDFREPOSITORY/001316.pdf
---
file: ../../../PDFREPOSITORY/001317.pdf
---
file: ../../../PDFREPOSITORY/001318.pdf
---
file: ../../../PDFREPOSITORY/001319.pdf
---
file: ../../../PDFREPOSITORY/001320.pdf
---
file: ../../../PDFREPOSITORY/001321.pdf
---
file: ../../../PDFREPOSITORY/001322.pdf
---
file: ../../../PDFREPOSITORY/001323.pdf
---
file: ../../../PDFREPOSITORY/001324.pdf
---
file: ../../../PDFREPOSITORY/001325.pdf
---
file: ../../../PDFREPOSITORY/001326.pdf
---
file: ../../../PDFREPOSITORY/001327.pdf
---
file: ../../../PDFREPOSITORY/001328.pdf
---
file: ../../../PDFREPOSITORY/001329.pdf
---
file: ../../../PDFREPOSITORY/001330.pdf
---
file: ../../../PDFREPOSITORY/001331.pdf
---
file: ../../../PDFREPOSITORY/001332.pdf
---
file: ../../../PDFREPOSITORY/001333.pdf
---
file: ../../../PDFREPOSITORY/001334.pdf
---
file: ../../../PDFREPOSITORY/001335.pdf
---
file: ../../../PDFREPOSITORY/001336.pdf
---
file: ../../../PDFREPOSITORY/001337.pdf
---
file: ../../../PDFREPOSITORY/001338.pdf
---
file: ../../../PDFREPOSITORY/001339.pdf
---
file: ../../../PDFREPOSITORY/001340.pdf
---
file: ../../../PDFREPOSITORY/001341.pdf
---
file: ../../../PDFREPOSITORY/001342.pdf
---
file: ../../../PDFREPOSITORY/001343.pdf
---
file: ../../../PDFREPOSITORY/001344.pdf
---
file: ../../../PDFREPOSITORY/001345.pdf
---
file: ../../../PDFREPOSITORY/001346.pdf
---
file: ../../../PDFREPOSITORY/001347.pdf
---
file: ../../../PDFREPOSITORY/001348.pdf
---
file: ../../../PDFREPOSITORY/001349.pdf
---
file: ../../../PDFREPOSITORY/001350.pdf
---
file: ../../../PDFREPOSITORY/001351.pdf
---
file: ../../../PDFREPOSITORY/001352.pdf
---
file: ../../../PDFREPOSITORY/001353.pdf
---
file: ../../../PDFREPOSITORY/001354.pdf
---
file: ../../../PDFREPOSITORY/001355.pdf
---
file: ../../../PDFREPOSITORY/001356.pdf
---
file: ../../../PDFREPOSITORY/001357.pdf
---
file: ../../../PDFREPOSITORY/001358.pdf
---
file: ../../../PDFREPOSITORY/001359.pdf
---
file: ../../../PDFREPOSITORY/001360.pdf
---
file: ../../../PDFREPOSITORY/001361.pdf
---
file: ../../../PDFREPOSITORY/001362.pdf
---
file: ../../../PDFREPOSITORY/001363.pdf
---
file: ../../../PDFREPOSITORY/001364.pdf
---
file: ../../../PDFREPOSITORY/001365.pdf
---
file: ../../../PDFREPOSITORY/001366.pdf
---
file: ../../../PDFREPOSITORY/001367.pdf
---
file: ../../../PDFREPOSITORY/001368.pdf
---
file: ../../../PDFREPOSITORY/001369.pdf
---
file: ../../../PDFREPOSITORY/001370.pdf
---
file: ../../../PDFREPOSITORY/001371.pdf
---
file: ../../../PDFREPOSITORY/001372.pdf
---
file: ../../../PDFREPOSITORY/001373.pdf
---
file: ../../../PDFREPOSITORY/001374.pdf
---
file: ../../../PDFREPOSITORY/001375.pdf
---
file: ../../../PDFREPOSITORY/001376.pdf
---
file: ../../../PDFREPOSITORY/001377.pdf
---
file: ../../../PDFREPOSITORY/001378.pdf
---
file: ../../../PDFREPOSITORY/001379.pdf
---
file: ../../../PDFREPOSITORY/001380.pdf
---
file: ../../../PDFREPOSITORY/001381.pdf
---
file: ../../../PDFREPOSITORY/001382.pdf
---
file: ../../../PDFREPOSITORY/001383.pdf
---
file: ../../../PDFREPOSITORY/001384.pdf
---
file: ../../../PDFREPOSITORY/001385.pdf
---
file: ../../../PDFREPOSITORY/001386.pdf
---
file: ../../../PDFREPOSITORY/001387.pdf
---
file: ../../../PDFREPOSITORY/001388.pdf
---
file: ../../../PDFREPOSITORY/001389.pdf
---
file: ../../../PDFREPOSITORY/001390.pdf
---
file: ../../../PDFREPOSITORY/001391.pdf
---
file: ../../../PDFREPOSITORY/001392.pdf
---
file: ../../../PDFREPOSITORY/001393.pdf
---
file: ../../../PDFREPOSITORY/001394.pdf
---
file: ../../../PDFREPOSITORY/001395.pdf
---
file: ../../../PDFREPOSITORY/001396.pdf
---
file: ../../../PDFREPOSITORY/001397.pdf
---
file: ../../../PDFREPOSITORY/001398.pdf
---
file: ../../../PDFREPOSITORY/001399.pdf
---
file: ../../../PDFREPOSITORY/001400.pdf
