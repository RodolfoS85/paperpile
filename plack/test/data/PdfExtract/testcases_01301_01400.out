---
authors: 'Suzuki, Masashi and Amano, Naoki and Kakinuma, Jun and Tateno, Masaru'
title: Use of a 3D Structure Data Base for Understanding Sequence-dependent Conformational Aspects of DNA
---
authors: 'Swain, Martin T. and Kemp, Graham J. L.'
title: A CLP Approach to the Protein Side-Chain Placement Problem
---
authors: 'Sweeney, Rosemary and Fan, Qichang and Yao, MENG-CHAO'
title: 'Antisense ribosomes: rRNA as a vehicle for antisense RNAs'
---
authors: 'Swisher, Jennifer and Duarte, Carlos M. and Su, Linhui Julie and Pyle, Anna Marie'
title: Visualizing the solvent-inaccessible core of a group II intron ribozyme
---
authors: 'Szymanski, Maciej and Barciszewska, Miroslawa Z. and Barciszewski, Jan and Erdmann, Volker A.'
title: 5S ribosomal RNA database Y2K
---
authors: 'Szymanski, Maciej and Barciszewska, Miroslawa Z. and Erdmann, Volker A. and Barciszewski, Jan'
title: 5S Ribosomal RNA Database
---
authors: 'Tabaska, Jack E. and Zhang, Michael Q.'
title: Detection of polyadenylation signals in human DNA sequences
---
authors: 'Tacke, Roland and Chen, Yan and Manley, James L.'
title: 'Sequence-specific RNA binding by an SR protein requires RS domain phosphorylation: Creation of an SRp40-specific splicing enhancer'
---
authors: 'Tadokoro, Keiko and Yamazaki-Inoue, Mayu and Tachibana, Maki and Fujishiro, Mina and Nagao, Kazuaki and Toyoda, Masashi and Ozaki, Miwako and Ono, Masami and Miki, Nobuhiro and Miyashita, Toshiyuki and Yamada, Masao'
doi: 10.1007/s10038-005-0261-9
title: 'Frequent occurrence of protein isoforms with or without a single amino acid residue by subtle alternative splicing: the case of Gln in DRPLA affects subcellular localization of the products'
---
authors: 'Tafer, Hakim and Ameres, Stefan L and Obernosterer, Gregor and Gebeshuber, Christoph A and Schroeder, Renee and Martinez, Javier and Hofacker, Ivo L'
doi: 10.1038/nbt1404
title: The impact of target site accessibility on the design of effective siRNAs
---
authors: 'Tailor, Chetankumar S. and Marin, Mariana and Nouri, Ali and Kavanaugh, Michael P. and Kabat, David'
doi: 10.1074/jbc.M100737200
title: Truncated Forms of the Dual Function Human ASCT2 Neutral Amino Acid Transporter/Retroviral Receptor Are Translationally Initiated at Multiple Alternative CUG and GUG Codons
---
authors: 'Takagi, Yasuomi and Warashina, Masaki and Stec, Wojciech J. and Yoshinari, Koichi and Taira, Kazunari'
title: Recent advances in the elucidation of the mechanisms of action of ribozymes
---
authors: 'Takeda, Tsutomu and Ono, Hirotaka and Sadakane, Kunihiko and Yamashita, Masafumi'
title: A Local Search Based Barrier Height Estimation Algorithm for DNA Molecular Transitions
---
authors: 'Takeuchi, Masayuki and Fujisawa, Hitoshi'
title: New alternatively spliced variants of calmodulin-dependent protein kinase II from rabbit liver
---
authors: 'Takusagawa, K. T. and Gifford, D. K.'
title: NEGATIVE INFORMATION FOR MOTIF DISCOVERY
---
authors: 'Tamada, Yoshinori and Kim, Sun Yong and Bannai, Hideo and Imoto, Seiya and Tashiro, Kousuke and Kuhara, Satoru and Miyano, Satoru'
title: Estimating Gene Networks from Gene Expression Data by Combining Bayesian Network Model with Promoter Element Detection
---
authors: 'Tanay, Amos and Shamir, Ron'
title: Computational Expansion of Genetic Networks
---
authors: 'Taneri, Bahar and Snyder, Ben and Novoradovsky, Alexey and Gaasterland, Terry'
title: Alternative splicing of mouse transcription factors affects their DNA-binding domain architecture and is tissue specific
---
authors: 'Tang, Chao'
title: Simple models of the protein folding problem
---
authors: 'Tang, Xinyu and Thomas, Shawna and Tapia, Lydia and Amato, Nancy M.'
title: Tools for Simulating and Analyzing RNA Folding Kinetics
---
authors: 'Tang, Jin and Breaker, Ronald R.'
title: Structural diversity of self-cleaving ribozymes
---
authors: 'Tang, W. and Gunn, T. M. and McLaughlin, D. F. and Barsh, G. S. and Schlossman, S. F. and Duke-Cohan, J. S.'
doi: 10.1073/pnas.110139897
title: Secreted and membrane attractin result from alternative splicing of the human ATRN gene
---
authors: 'Tang, Xinyu and Kirkpatrick, Bonnie and Thomas, Shawna and Song, Guang and Amato, Nancy M.'
title: Using Motion Planning to Study RNA Folding Kinetics
---
authors: 'Tavazoie, Saeed and Hughes, Jason D. and Campbell, Michael J. and Cho, Raymond J. and Church, George M.'
title: Systematic determination of genetic network architecture
---
authors: 'Taverna, D. M. and Goldstein, R. M.'
title: THE EVOLUTION OF DUPLICATED GENES CONSIDERING PROTEIN STABILITY CONSTRAINTS
---
authors: 'Tay, Yvonne and Zhang, Jinqiu and Thomson, Andrew M. and Lim, Bing and Rigoutsos, Isidore'
doi: 10.1038/nature07299
title: 'MicroRNAs to Nanog, Oct4 and Sox2 coding regions modulate embryonic stem cell differentiation'
---
authors: 'Taylor, Ronald C. and Acquaah-Mensah, George and Singhal, Mudita and Malhotra, Deepti and Biswal, Shyam'
doi: 10.1371/journal.pcbi.1000166
title: Network Inference Algorithms Elucidate Nrf2 Regulation of Mouse Lung Oxidative Stress
---
authors: 'Taylor, Martin S. and Ponting, Chris P. and Copley, Richard R.'
doi: 10.1101/gr.1977804
title: Occurrence and Consequences of Coding Sequence Insertions and Deletions in Mammalian Genomes
---
authors: 'Thadani, Rahul and Tammi, Martti T'
doi: 10.1186/1471-2105-7-S5-S20
title: 'MicroTar: predicting microRNA targets from RNA duplexes'
---
authors: 'Thanaraj, T. A.'
title: Positional characterisation of false positives from computational prediction of human splice sites
---
authors: 'Thanbichler, Martin and Bock, August and Goody, Roger S.'
doi: 10.1074/jbc.M002496200
title: Kinetics of the Interaction of Translation Factor SelB from Escherichia coli with Guanosine Nucleotides and Selenocysteine Insertion Sequence RNA
---
authors: 'Thanbichler, Martin and Bock, August'
title: The function of SECIS RNA in translational control of gene expression in Escherichia coli
---
authors: 'Thastrom, A. and Lowary, P. T. and Widlund, H. R. and Cao, H. and Kubista, M. and Widom, J.'
title: Sequence Motifs and Free Energies of Selected Natural and Non-natural Nucleosome Positioning DNA Sequences
---
authors: 'Thayer, Kelly M. and Beveridge, D. L.'
doi: 10.1073/pnas.132148699
title: Hidden Markov models from molecular dynamics simulations on DNA
---
authors: 'Thbault, Patricia and de Givry, Simon and Schiex, Thomas and Gaspin, Christine'
title: Combining constraint processing and pattern matching to describe and locate structured motifs in genomic sequences
---
authors: 'Theissen, Günter and Becker, Annette and Di Rosa, Alexandra and Kanno, Akira and Kim, Jan T. and Münster, Thomas and Winter, Kai-Uwe and Saedler, Heinz'
title: A short history of MADS-box genes in plants
---
authors: 'Theodosiou, T. and Darzentas, N. and Angelis, L. and Ouzounis, C. A.'
title: 'PuReD-MCL: a graph-based PubMed document clustering methodology'
---
authors: 'Thieffry, Denis and Huerta, Araceli M. and Perez-Rueda, Ernesto and Collado-Vides, Julio'
title: 'From specific gene regulation to genomic networks: a global analysis of transcriptional regulation in Escherichia coli'
---
authors: 'Thiele, Ralf and Zimmer, Ralf and Lengauer, Thomas'
title: Protein Threading by Recursive Dynamic Programming
---
authors: 'Thijs, Gert and Lescot, Magali and Marchal, Kathleen and Rombauts, Stephane and De Moor, Bart and Rouze, Pierre and Moreau, Yves'
title: A higher-order background model improves the detection of promoter regulatory elements by Gibbs sampling
---
authors: 'Thijs, Gert and Marchal, Kathleen and Lescot, Magali and Rombauts, Stephane and DE MOOR, Bart and ROUZÉ, Pierre and Moreau, Yves'
title: A Gibbs Sampling Method to Detect Overrepresented Motifs in the Upstream Regions of Coexpressed Genes
---
authors: 'Thompson, Julie D. and Plewniak, Frédéric and Poch, Olivier'
title: A comprehensive comparison of multiple sequence alignment programs
---
authors: 'Tian, Bin and Pan, Zhenhua and Lee, Ju Youn'
doi: 10.1101/gr.5532707
title: Widespread mRNA polyadenylation events in introns indicate dynamic interplay between polyadenylation and splicing
---
authors: 'Tiana, Guido and Shakhnovich, Boris E. and Dokholyan, Nikolay V. and Shakhnovich, Eugene I.'
doi: 10.1073/pnas.0306638101
title: Imprint of evolution on protein structures
---
authors: 'Todd, Annabelle E and Orengo, Christine A and Thornton, Janet M'
title: 'Evolution of protein function, from a structural perspective'
---
authors: 'Tokuriki, Nobuhiko and Stricher, Francois and Serrano, Luis and Tawfik, Dan S.'
doi: 10.1371/journal.pcbi.1000002
title: How Protein Stability and New Functions Trade Off
---
authors: 'Tollervey, David and Kiss, Tamas'
title: Function and synthesis of small nucleolar RNAs
---
authors: 'Tomita, Masaru and Shimizu, Nobuyoshi and Brutlagt, Douglas L.'
title: 'Introns and Reading Frames: Correlation Between Splicing Sites and Their Codon Positions'
---
authors: 'Tompa, Martin and Li, Nan and Bailey, Timothy L and Church, George M and De Moor, Bart and Eskin, Eleazar and Favorov, Alexander V and Frith, Martin C and Fu, Yutao and Kent, W James and Makeev, Vsevolod J and Mironov, Andrei A and Noble, William Stafford and Pavesi, Giulio and Pesole, Graziano and Régnier, Mireille and Simonis, Nicolas and Sinha, Saurabh and Thijs, Gert and van Helden, Jacques and Vandenbogaert, Mathias and Weng, Zhiping and Workman, Christopher and Ye, Chun and Zhu, Zhou'
doi: 10.1038/nbt1053
title: Assessing computational tools for the discovery of transcription factor binding sites
---
authors: 'Torgersen, Jacob and Nourizadeh-Lillabadi, Rasoul and Husebye, Harald and Aleström, Peter'
title: In silico and in situ characterization of the zebrafish (Danio rerio) gnrh3 (sGnRH) gene
---
authors: 'Tormay, Peter and Bock, August'
title: Barriers to Heterologous Expression of a Selenoprotein Gene in Bacteria
---
authors: 'Townson, Steven M. and Dobrzycka, Klaudia M. and Lee, Adrian V. and Air, Mamie and Deng, Wanleng and Kang, Kaiyan and Jiang, Shiming and Kioka, Noriyuki and Michaelis, Kai and Oesterreich, Steffi'
doi: 10.1074/jbc.M212988200
title: 'SAFB2, a New Scaffold Attachment Factor Homolog and Estrogen Receptor Corepressor'
---
authors: 'Trainor, Cecelia D. and Ghirlando, Rodolfo and Simpson, Melanie A.'
doi: 10.1074/jbc.M000020200
title: GATA Zinc Finger Interactions Modulate DNA Binding and Transactivation
---
authors: 'Tramontano, Anna'
title: Of men and machines
---
authors: 'Travers, Andrew A.'
doi: 10.1038/sj.embor.embor741
title: 'Priming the nucleosome: a role for HMGB proteins?'
---
authors: 'Trethewey, Richard N'
title: Gene discovery via metabolic profiling
---
authors: 'Trueb, Beat and Zhuang, Lei and Taeschler, Sara and Wiedemann, Markus'
doi: 10.1074/jbc.M300281200
title: 'Characterization of FGFRL1, a Novel Fibroblast Growth Factor (FGF) Receptor Preferentially Expressed in Skeletal Tissues'
---
authors: 'Trusina, Ala and Sneppen, Kim and Dodd, Ian B. and Shearwin, Keith E. and Egan, J. Barry'
doi: 10.1371/journal.pcbi.0010074
title: 'Functional Alignment of Regulatory Networks: A Study of Temperate Phages'
---
authors: 'Tsai, CHUNG-JUNG and Kumar, Sandeep and Ma, Buyong and Nussinov, Ruth'
doi: 10.1110/ps.8.6.1181
title: 'Folding funnels, binding funnels, and protein function'
---
authors: 'Tujebajeva, Rosa M. and Copeland, Paul R. and Xu, Xue-Ming and Carlson, Bradley A. and Harney, John W. and Driscoll, Donna M. and Hatfield, Dolph L. and Berry, Marla J.'
title: Decoding apparatus for eukaryotic selenocysteine insertion
---
authors: 'Tullai, John W. and Schaffer, Michael E. and Mullenbrock, Steven and Kasif, Simon and Cooper, Geoffrey M.'
title: Identification of Transcription Factor Binding Sites Upstream of Human Genes Regulated by the Phosphatidylinositol 3-Kinase and MEK/ERK Signaling Pathways
---
authors: 'Tusnady, Gabor E. and Simon, Istvan'
title: 'Principles Governing Amino Acid Composition of Integral Membrane Proteins: Application to Topology Prediction'
---
authors: 'Tusnady, Gabor E. and Simon, Istvan'
title: The HMMTOP transmembrane topology prediction server
---
authors: 'Tyers, Mike and Mann, Matthias'
title: From genomics to proteomics
---
authors: 'Uetz, Peter and Pohl, Ehmke'
title: 'Protein-Protein- und Protein-DNA-Interaktionen'
---
authors: 'Ule, Jernej and Stefani, Giovanni and Mele, Aldo and Ruggiu, Matteo and Wang, Xuning and Taneri, Bahar and Gaasterland, Terry and Blencowe, Benjamin J. and Darnell, Robert B.'
doi: 10.1038/nature05304
title: An RNA map predicting Nova-dependent splicing regulation
---
authors: 'Ulitsky, Igor and Shamir, Ron'
doi: 10.1038/msb4100144
title: Pathway redundancy and protein essentiality revealed in the Saccharomyces cerevisiae interaction networks
---
authors: 'Unger, Ron and Moult, John'
title: Local Interactions Dominate Folding in a Simple Protein Model
---
authors: 'Unoson, Cecilia and Wagner, E. Gerhart H.'
doi: 10.1111/j.1365-2958.2008.06416.x
title: A small SOS-induced toxin is targeted against the inner membrane in Escherichia coli
---
authors: 'Urano, Aki and Suzuki, Miho M. and Zhang, Peijun and Satoh, Nori and Satoh, Gouki'
title: Expression of muscle-related genes and two MyoD genes during amphioxus notochord development
---
authors: 'Urban, Johannes H. and Vogel, Jorg'
doi: 10.1371/journal
title: Two Seemingly Homologous Noncoding RNAs Act Hierarchically to Activate glmS mRNA Translation
---
authors: 'Ureta-Vidal, Abel and Ettwiller, Laurence and Birney, Ewan'
doi: 10.1038/nrg1043
title: 'COMPARATIVE GENOMICS: GENOME-WIDE ANALYSIS IN METAZOAN EUKARYOTES'
---
authors: 'Vagner, Stphan and Touriol, Christian and Galy, Bruno and Audigier, Sylvie and Gensac, Marie-Claire and Amalric, Francois and Bayard, Francis and Prats, Herv and Prats, Anne-Catherine'
title: 'Translation of CUG- but Not AUG-initiated Forms of Human Fibroblast Growth Factor 2 is Activated in Transformed and Stressed Cells'
---
authors: 'Valdivia-Granda, Willy'
title: The next meta-challenge for Bioinformatics
---
authors: 'Valentin-Hansen, Poul and Eriksen, Maiken and Udesen, Christina'
doi: 10.1046/j.1365-2958.2003.03935.x
title: 'The bacterial Sm-like protein Hfq: a key player in RNA transactions'
---
authors: 'van Batenburg, F. H. D. and Gultyaev, A. P. and Pleij, C. W. A. and Ng, J. and Oliehoek, J.'
title: 'PseudoBase: a database with RNA pseudoknots'
---
authors: 'van Batenburg, F. H. D. and Gultyaev, A. P. and Pleij, C. W. A.'
title: 'PseudoBase: structural information on RNA pseudoknots'
---
authors: 'Vandenbroucke, Ina and Callens, Tom and De Paepe, Anne and Messiaen, Ludwine'
title: Complex splicing pattern generates great diversity in human NF1 transcripts
---
authors: 'Vandepoele, Klaas and Casneuf, Tineke and Van de Peer, Yves'
doi: 10.1186/gb-2006-7-11-r103
title: Identification of novel regulatory modules in dicotyledonous plants using expression data and comparative genomics
---
authors: 'Vanderpool, Carin K. and Gottesman, Susan'
doi: 10.1111/j.1365-2958.2004.04348.x
title: Involvement of a novel transcriptional activator and small RNA in post-transcriptional regulation of the glucose phosphoenolpyruvate phosphotransferase system
---
authors: 'Vanderpool, Carin K and Gottesman, Susan'
title: Noncoding RNAs at the membrane
---
authors: 'van Helden, J. and Andre, B. and Collado-Vides, J.'
title: Extracting Regulatory Sites from the Upstream Region of Yeast Genes by Computational Analysis of Oligonucleotide Frequencies
---
authors: 'van Helden, Jacques and Rios, Alma. F. and Collado-Vides, Julio'
title: Discovering regulatory elements in non-coding sequences by analysis of spaced dyads
---
authors: 'Varma, Sudhir and Simon, Richard'
doi: 10.1186/1471-2105-5-126
title: Iterative class discovery and feature selection using Minimal Spanning Trees
---
authors: 'Vecerek, Branislav and Moll, Isabella and Afonyushkin, Taras and Kaberdin, Vladimir and Bläsi, Udo'
doi: 10.1046/j.1365-2958.2003.03727.x
title: 'Interaction of the RNA chaperone Hfq with mRNAs: direct and indirect roles of Hfq in iron metabolism of Escherichia coli'
---
authors: 'Vecerek, Branislav and Moll, Isabella and Blasi, Udo'
doi: 10.1038/sj.emboj.7601553
title: Control of Fur synthesis by the non-coding RNA RyhB and iron-responsive decoding
---
authors: 'Veenman, C. J. and Reinders, M. J. T. and Backer, E.'
title: A Maximum Variance Cluster Algorithm
---
authors: 'Veerla, Srinivas and Höglund, Mattias'
doi: 10.1186/1471-2105-7-384
title: Analysis of promoter regions of co-expressed genes identified by microarray analysis
---
authors: 'Venables, Julian P.'
title: Aberrant and Alternative Splicing in Cancer
---
authors: 'Venables, Julian P'
title: Alternative splicing in the testes
---
authors: 'Vickers, Timothy A. and Wyatt, Jacqueline R. and Freier, Susan M.'
title: Effects of RNA secondary structure on cellular antisense activity
---
authors: 'V¯ksna, Juris and Gilbert, David'
title: Pattern Matching and Pattern Discovery Algorithms for Protein Topologies
---
authors: 'Vitkup, Dennis and Sander, Chris and Church, George M'
title: The amino-acid mutational spectrum of human genetic disease
---
authors: 'Vogel, Jorg and Argaman, Liron and Wagner, E. Gerhart H. and Altuvia, Shoshy'
doi: 10.1016/j.cub.2004.12.003
title: The Small RNA IstR Inhibits Synthesis of an SOS-Induced Toxic Peptide
---
authors: 'Vogel, Jorg and Papenfort, Kai'
doi: 10.1016/j.mib.2006.10.006
title: Small non-coding RNAs and the bacterial outer membrane
---
authors: 'Vogel, Jörg'
doi: 10.1111/j.1365-2958.2008.06505.x
title: A rough guide to the non-coding RNA world of Salmonella
---
authors: 'Vogel, Jorg and Sharma, Cynthia Mira'
doi: 10.1515/BC.2005.140
title: How to find small non-coding RNAs in bacteria
---
authors: 'Voight, Benjamin F. and Kudaravalli, Sridhar and Wen, Xiaoquan and Pritchard, Jonathan K.'
doi: 10.1371/journal.pbio.0040072
title: A Map of Recent Positive Selection in the Human Genome
---
authors: 'Volfovsky, Natalia and Haas, Brian J. and Salzberg, Steven L.'
doi: 10.1101/gr.677503
title: Computational Discovery of Internal Micro-Exons
---
authors: 'Volz, Armin and Ehlers, Anke and Younger, Ruth and Forbes, Simon and Trowsdale, John and Schnorr, Dietmar and Beck, Stephan and Ziegler, Andreas'
doi: 10.1074/jbc.M212424200
title: Complex Transcription and Splicing of Odorant Receptor Genes
